app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -23,12 +122,22 @@ export const CollectionsPage = ({ data }) => {
     >
       <Box
         sx={{
+          maxWidth: "730px",
```

Comment:
Where did you get this number?

Reply:
I used the width from the description on the division landing page: 
https://github.com/NYPL/digital-collections/pull/193/files#diff-78d2323f3e142cea5e900f6080991f164b2ecea631e8d0b5e9b64c1490f9f7dfR41